README.md

```diff
@@ -38,29 +37,67 @@ You can use `klaemo/couchdb-ssl` as the base image for your own couchdb instance
 You might want to provide your own version of the following files:
 
 * `local.ini` for CouchDB.
-* `stud.conf` for Stud configuration.
-* `stud.pem` is the (you guessed it) pem file.
+* `nginx.conf` for nginx configuration.
+* `server.crt` for nginx configuration (certificate).
+* `server.key` for nginx configuration (private key).
+* `dhparams.pem` for nginx configuration (diffie-helman parameters).
+* `chain.pem` for nginx configuration (all three above combined).
 
 Example Dockerfile:
-```
+
+```bash
 FROM klaemo/couchdb-ssl
 
 ADD local.ini /usr/local/etc/couchdb/
-ADD stud.conf /usr/local/etc/stud/
-ADD stud.pem /usr/local/etc/stud/
+ADD nginx.conf /etc/nginx/
+ADD server.crt /etc/nginx/certs/
+ADD server.key /etc/nginx/certs/
+ADD dhparams.pem /etc/nginx/certs/
+ADD chain.pem /etc/nginx/certs/
```

Comment:
You probably didn't commit `chain.pem`, because it doesn't exist for me. Maybe we can create it, instead of adding it?

`RUN cd /etc/nginx/certs && cat server.key server.crt dhparams.pem > chain.pem`